### PR TITLE
Fix style discrepancies in Firefox

### PR DESCRIPTION
**Pull request by DingoEatingFuzz**
- Spacer in the top-right menu are now aligned with text
- Allocation charts are vertically centered
- Allocation charts use the full width available

### CODE REVIEW

File: ui/app/mixins/window-resizable.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+
+const { run, $ } = Ember;
+
+export default Ember.Mixin.create({
```
Comment (meirish) [inline ui/app/mixins/window-resizable.js]: I know we don't have a lot of components using this now (and maybe we won't ever?) - but would this be better as (or paired with) a service that had an array of handlers that would lazily attach the global resize handler? That way there's just one resize event handler that fires all of attached callbacks (shout out to $.Callbacks). Also this is 100% something we could do later, just wanted to put a bug in your ear about it.

Comment (DingoEatingFuzz) [reply ui/app/mixins/window-resizable.js]: > That way there's just one resize event handler that fires all of attached callbacks

Isn't that what `jQuery.on` is already doing?

Comment (meirish) [reply ui/app/mixins/window-resizable.js]: For posterity: jQuery _is_ already doing this! - @DingoEatingFuzz went spelunking and shared this:   https://github.com/jquery/jquery/blob/master/src/event.js#L184-L216

**Comment by DingoEatingFuzz** [comment]
I haven't given it thought before, but it doesn't seem so useful for charts. Mostly because little details like corner radius, element spacing, and font size shouldn't scale.

**Pull request merged**